app/apphandlers/gatewayAppHandler.go

```diff
@@ -132,23 +132,26 @@ func (a *gatewayApp) StartRudderCore(ctx context.Context, options *app.Options)
 	})
 	drainConfigManager, err := drain_config.NewDrainConfigManager(config, a.log.Child("drain-config"))
 	if err != nil {
-		return fmt.Errorf("drain config manager setup: %v", err)
+		a.log.Errorw("drain config manager setup fail", "error", err)
```

Comment:
So in case of an error, gateway will be operating in degraded mode, at least with respect to draining jobs. Do we plan to setup a P0 alert for this?

Reply:
yes we will setup an alert for this

Reply:
It would be nice to have the relevant pr mentioned here :)

Reply:
https://github.com/rudderlabs/rudder-devops/pull/9820